sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/pyproject.toml

```diff
@@ -1,3 +1,51 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "wheel"]  # Requires 61.0.0 for dynamic version
```

Comment:
nit: `setuptools>=61.0.0` here, but `setuptools>=77.0.3` elsewhere. Why the difference?

Reply:
good catch